src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -653,6 +655,14 @@ public void AnalyzeCommandBase_DefaultEndToEndAnalysis()
             configurationNotificationCount.Should().Be(0);
         }
 
+        [Fact]
+        public void AnalyzeCommandBase_EndToEndAnalysisWithPostUri()
+        {
+            PostUriTestHelper(@"https://httpbin.org/post", TestAnalyzeCommand.SUCCESS, RuntimeConditions.None);
```

Comment:
>@"https://httpbin.org/post" [](http://example.com/codeflow?start=30&length=27)

have a question for these 2,
these are public test uri and we only post a sample Sarif without any secret.
let me know if this is ok.

Reply:
This kind of off-network activity can cause issues in unit tests. Typically we would mock this. Ed has a nice mockable HTTP client.